rust/osm2lanes/src/transform/lanes_to_tags.rs

```diff
@@ -16,6 +16,12 @@ pub struct LanesToTagsConfig {
     pub check_roundtrip: bool,
 }
 
+impl LanesToTagsConfig {
+    pub fn new(check_roundtrip: bool) -> Self {
+        LanesToTagsConfig { check_roundtrip }
+    }
+}
+
```

Comment:
needed due to `#[non_exhaustive]`, but if this gets much longer it will probably want a builder...